sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -26,36 +25,38 @@ def _check_boundary_response_method(estimator, response_method):
         If set to 'auto', the response method is tried in the following order:
         :term:`decision_function`, :term:`predict_proba`, :term:`predict`.
 
+    class_of_interest : int, float, bool, str or None
+        The class considered when plotting the decision. If the label is specified, it
+        then possible to plot the decision boundary in multiclass settings.
+
+        .. versionadded:: 1.4
+
     Returns
     -------
-    prediction_method: callable
-        Prediction method of estimator.
+    prediction_method : list of str or str
+        The name of the response methods to use or a list of such names.
     """
     has_classes = hasattr(estimator, "classes_")
     if has_classes and _is_arraylike_not_scalar(estimator.classes_[0]):
         msg = "Multi-label and multi-output multi-class classifiers are not supported"
         raise ValueError(msg)
 
     if has_classes and len(estimator.classes_) > 2:
-        if response_method not in {"auto", "predict"}:
+        if response_method not in {"auto", "predict"} and class_of_interest is None:
             msg = (
                 "Multiclass classifiers are only supported when response_method is"
-                " 'predict' or 'auto'"
+                " 'predict' or 'auto', or you must provide `class_of_interest` to "
+                " select a specific class to plot the decision boundary."
```

Comment:
"Multiclass classifiers are only supported when `response_method`"
                " is 'predict' or 'auto'. Else you must provide `class_of_interest`"
                " to plot the decision boundary of a specific class."